manifests/config.pp

```diff
@@ -193,28 +193,22 @@
   }
 
   case $facts['os']['family'] {
+    'Archlinux': {
+      $has_systemd = true
```

Comment:
Can we perhaps make use of the systemd fact in camptopcamp/systemd module?

Reply:
I don't know much about that fact.. if it works accurately it would be cool to use. I can add the module as a dependency? It would also be useful for the systemctl daemon reload stuff.

Reply:
yes, you can add it as a dependency to the metadata.json and use the fact